src/library.js

```diff
@@ -364,6 +364,7 @@ mergeInto(LibraryManager.library, {
   // module scope: the built-in runtime function abort(), and this library
   // function _abort(). Remove one of these, importing two functions for the
   // same purpose is wasteful.
+  abort__proxy: 'sync',
```

Comment:
This seems like it should be its own change.  (along with the change to knownHandlers).